Log in form - HTML and Tailwind CSS

kristinatonnessen opened the pull request:
CHANGES IN PR - I've added the css issue to this pull request.

### What does this PR do?

This pull request addresses the login (HTML) issue and the Login (CSS) issue. I've finalized the Log in form including both HTML structure and Tailwind CSS styling. I've also installed Tailwind v4 in the branch, fixing the tailwind issues.

### Description of Task to be completed?

- Created the Log in form using semantic HTML
- Styled the form with Tailwind CSS, ensuring responsive and accessible design.
- Resolved Tailwind version issue, allowing proper styling using custom colors and fonts.

### How should this be manually tested?

Clone the repository and navigate to the branch for this pull request : login-page-ingelinn
Open the HTML file login-page.html to verify that:

- The page structure follows proper semantic guidelines.
- The Log in form renders correctly and maintains proper styling on different screen sizes.
- The form fields, labels and button are aligned and styled as expected.
- The form is fully responsive, adapting to both mobile and desktop views.
- The Tailwind classes are applied correctly, confirming that the CSS is working as expected.

### Any background context you want to provide?

Previously, there were issues with Tailwind CSS in the project, requiring work in a separate test project and not being able to use the customized styling. Have in mind that I've had to change the tailwind classes after tailwind v4 was fixed in edugate, and be aware of the possibility I have missed some classes that should have been changed.

kristinatonnessen committed:
copied ingelinns code to new branch

Tomatsuppe11211 reviewed (dismissed):
Looks good. I am looking forward to seeing more Tailwind!

hallsi90 committed:
Update login-page.html

Tomatsuppe11211 reviewed (dismissed):
Looking good!

hallsi90 committed:
Added styling

Finished styling the form
Made some small improvements on the html form

hallsi90 committed:
Merge branch 'develop' into login-page-ingelinn

hallsi90 committed:
Updated log in page

Installed tailwind v4.
Made minor changes to the html form.
Changed styling classes to fit custom colors and fonts.
Made form responsive.

hallsi90 commented:
CHANGES IN PR - I've added the css issue to this pull request -> see description.

Tomatsuppe11211 reviewed (changes requested):
Quick fix:

line 69: Button is supposed to have a bold font

Design changes: 
The form is not supposed to have a border on the mobile version


recommended: 
If you want to style the forms to look more like the design you can check out this link: 
https://tailwindcss.com/docs/box-shadow

hallsi90 committed:
Update login-page.html

Updated form colors and shadow.
Removed form border on small screen sizes.

hallsi90 committed:
Merge branch 'develop' into login-page-ingelinn

hallsi90 commented:
- I've left the button with the classes .primary-button and .primary-button:hover since we agreed on that at our team meeting
- I have removed the form border on smaller screen sizes
- I have updated the border shadows
- I have updated the border and shadow color to #cccccc (like it is in the figma design)

myrset reviewed (approved):
Good job on the changes. The login-page looks good.

Tomatsuppe11211 reviewed (changes requested):
Looking good. There are some quick fixes here tho:

1) button on the mobile version has to be smaller. Try adding w-... or/and h-... in the class. That should help you resize without ruining the buttons style

2) the input fields need to be wider (see images)

Your code:
![image](https://github.com/user-attachments/assets/1d5ae238-c7bb-4442-9de5-adb631d7e6eb)


Figma design:
![image](https://github.com/user-attachments/assets/a644cda0-40c5-4058-b246-c3cf870b7a67)

Janicke92 committed:
Merge branch 'develop' into login-page-ingelinn

Tomatsuppe11211 committed:
Merge branch 'develop' into login-page-ingelinn
